proxies/python/app.py

```diff
@@ -28,4 +29,31 @@ def spec():
         "name": "Python",
         "version": "", # TODO add branch name or SDK version here
         "capabilities": ["EdgeDB", "CloudBucketing"]
-    }
\ No newline at end of file
+    }
+
+@app.post("/client")
+def client():
+    body = request.get_json()
+    client_id, sdk_key, options = [body.get(k, None) for k in ('clientId', 'sdkKey', 'options')]
```

Comment:
should we move this to its own file, like in nodejs or keep in same?

Reply:
I might want to do that later, it does not seem super straightforward to organize flask into multiple files :/

Reply:
actually I figured this out and did it in a later PR on the stack